common/build.gradle

```diff
@@ -19,6 +19,10 @@ dependencies {
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '4.4.0'
 }
 
+lombok {
```

Comment:
Just curious: should we remove this: `id "io.freefair.lombok"`?

Reply:
> Just curious: should we remove this: `id "io.freefair.lombok"`?

The plugin is still needed, but it allows to override the lombok version (this is taken from the plugin docs)